account_journal_general_sequence/models/account_move.py

```diff
@@ -61,7 +61,9 @@ def _compute_entry_number(self):
         # Cache all the new numbers to avoid wasting recomputations, caused by
         # searches done by _next() in the loop below
         chosen_map = {}
-        for move in chosen.sorted(lambda one: (one.date, one.name, one.id)):
+        for move in chosen.sorted(
+            lambda one: (one.date or "", one.name or "", one.id or 0)
```

Comment:
If date is False or None (this shouldn't happen) will raise TypeError between datetime and empty string

```suggestion
            lambda one: (one.date and one.date.strftime(DEFAULT_SERVER_DATE_FORMAT) or "", one.name or "", one.id or 0)
```

Reply:
Oops already merged! Well, I'll push the fix if it happens.